packages/flutter_tools/templates/module/ios/host_app_ephemeral/Runner.xcodeproj.tmpl/project.pbxproj.tmpl

```diff
@@ -437,6 +443,9 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
+{{#hasIosDevelopmentTeam}}
+				DEVELOPMENT_TEAM = {{iosDevelopmentTeam}};
```

Comment:
Just curious, what happens for `DEVELOPMENT_TEAM = "";`, would it be cleaner that DEVELOPMENT_TEAM is set to an empty string if the developer didn't set it?